apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -61,7 +61,7 @@ function showHomepage() {
     unitId = parseInt(query.unitId, 10);
     updateQueryParam('unitId', undefined, true);
   }
-  if (courseOfferingId && courseVersionId) {
+  if ((courseOfferingId && courseVersionId) || query.openAddSectionDialog) {
```

Comment:
it's probably worth doing something like `updateQueryParam('openAddSectionDialog', undefined, true)` to remove the flag from the url after hitting the home screen. That way it won't constantly be opening for people because of saved browser history